### PR TITLE
[New][Author] Add react-uwp w/ git auto-update via single package.json 

### LINKED ISSUE #11448: [Request][Author] Add react-uwp
**Library name:** react-uwp
**Git repository url:** https://github.com/myxvisual/react-uwp
**npm package name or url** (if there is one): react-uwp
**License (List them all if it's multiple):** MIT
**Official homepage:** https://www.react-uwp.com
**Wanna say something? Leave message here:** Because it involves webfont and browser support, I hope it can be added to cdnjs, for developers to simply run on the browser, thanks~

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by myxvisual**
Add react-uwp via single package.json w/ git auto-update
Close #11448

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):  https://github.com/myxvisual/react-uwp
 * Official website (optional, not the repository): https://www.react-uwp.com
 * NPM package url (optional): https://www.npmjs.com/package/react-uwp
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 7
   - Count of stars: 122
   - Count of forks: 6
 * NPM download stats (optional):
   - Downloads in the last day: 56
   - Downloads in the last week: 279
   - Downloads in the last month: 370

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by myxvisual** [comment]
hi~  @sufuf3 ,can you help review code change ?
Not clear why the CI test is still failed...
i temporarily turn off this PR.

File: ajax/libs/react-uwp/READEME.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# [react-uwp-bower](https://github.com/myxvisual/react-uwp-bower)
```
Comment (PeterDaveHello) [inline ajax/libs/react-uwp/READEME.md]: Please remove this file.

File: ajax/libs/react-uwp/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "react-uwp",
+  "version": "1.0.8",
```
Comment (PeterDaveHello) [inline ajax/libs/react-uwp/package.json]: "version" should be removed as you didn't commit its version

**Comment by PeterDaveHello** [comment]
@myxvisual I'm sorry but looks like this library didn't match our minimal popularity requirement to host it, point me if I'm wrong :)

**Comment by PeterDaveHello** [comment]
@myxvisual would you please fill the library info in comments first?

File: ajax/libs/react-uwp/READEME.md (hunk 1 above)
Comment (myxvisual) [inline ajax/libs/react-uwp/READEME.md]: sure

**Comment by myxvisual** [comment]
@PeterDaveHello I need thirty minutes to do something, thanks~

**Comment by PeterDaveHello** [comment]
@myxvisual No worries at all, take your time.

**Comment by myxvisual** [comment]
@PeterDaveHello thanks~

**Comment by myxvisual** [comment]
@PeterDaveHello This PR is too much commit, should i pull anotther new request?

File: ajax/libs/react-uwp/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      {
+        "basePath": "",
+        "files": [
+          "**/*.js",
+          "**/*.css",
+          "**/*.svg",
+          "**/*.ttf",
+          "**/*.woff",
+          "**/*.woff2",
+          "**/*.eot"
```
Comment (sufuf3) [inline ajax/libs/react-uwp/package.json]: It's useful to us to let the code clear, would you modify from A to B?
- A
```
          "**/*.js",
          "**/*.css",
          "**/*.svg",
          "**/*.ttf",
          "**/*.woff",
          "**/*.woff2",
          "**/*.eot"
```

- B
```
          "**/*.+(js|css|svg|ttf|woff|woff2|eot)"
```
(This matches one or more occurences of the patterns provided.)

And do you want to add `*png` files?

**Comment by PeterDaveHello** [comment]
No need, I can squash them.

File: ajax/libs/react-uwp/package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        ]
+      }
+    ]
+  },
+  "namespace": "ReactUWP"
```
Comment (myxvisual) [inline ajax/libs/react-uwp/package.json]: sure~ thanks

File: ajax/libs/react-uwp/package.json (hunk 3 above)
Comment (myxvisual) [inline ajax/libs/react-uwp/package.json]: nice~ thanks!

**Comment by myxvisual** [comment]
@sufuf3 library info is updated, package.json is also updated, thanks~

File: ajax/libs/react-uwp/package.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  "author": "myxvisual",
+  "license": "MIT",
+  "homepage": "https://www.react-uwp.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/myxvisual/react-uwp-bower.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": ["**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jp?eg)"]
```
Comment (sufuf3) [inline ajax/libs/react-uwp/package.json]: `jp?eg` this part is a little bit weird. Maybe we could use `jpeg|jpg` to match the patterns.

Would you modify from A to B?
- A
```
    "fileMap": [
      {
        "basePath": "",
        "files": ["**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jp?eg)"]
      }
    ]
```

- B
```
    "fileMap": [
      {
        "basePath": "",
        "files": [
          "**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jpeg|jpg)"
        ]
      }
    ]
```

Thank you.

Comment (myxvisual) [inline ajax/libs/react-uwp/package.json]: 谢谢妹纸...

**Comment by myxvisual** [comment]
@sufuf3 the package.json is updated.

File: ajax/libs/react-uwp/package.json [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  "license": "MIT",
+  "homepage": "https://www.react-uwp.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/myxvisual/react-uwp-bower.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jpg|jpeg)"
```
Comment (PeterDaveHello) [inline ajax/libs/react-uwp/package.json]: Maybe just `!(LICENSE)` can be much more easier and future compatible? cc @cdnjs/intern3 @cdnjs/library-reviewer

Comment (myxvisual) [inline ajax/libs/react-uwp/package.json]: Sure, thanks for the advice.

File: ajax/libs/react-uwp/package.json [hunk 7]
```diff
@@ -28,7 +28,7 @@
       {
         "basePath": "",
         "files": [
-          "**/*.+(js|css|svg|ttf|woff|woff2|eot|png|jpg|jpeg)"
+          "!(LICENSE)"
```
Comment (PeterDaveHello) [inline ajax/libs/react-uwp/package.json]: Sorry that I didn't say it clear enough, the whole line should be `"**/!(LICENSE)"`

Comment (myxvisual) [inline ajax/libs/react-uwp/package.json]: Hi, the filnes is updated, thanks~

**Comment by PeterDaveHello** [comment]
@myxvisual https://github.com/myxvisual/react-uwp-bower seems dated? I saw the latest version on src repo and npm is v1.0.9 but in the bower repo is v1.0.8

**Comment by myxvisual** [comment]
@PeterDaveHello thanks~ the bower repo is updated.

**Comment by PeterDaveHello** [comment]
@myxvisual looks like our bot just picked it up: https://github.com/cdnjs/cdnjs/commit/d363979c7c2bd3db6f0a9e92da9229b91fb916ae, any chance to make the bower repo update automatically?

**Comment by myxvisual** [comment]
@PeterDaveHello sure, i wiil add automatically to release script.

**Comment by PeterDaveHello** [comment]
@myxvisual thanks

**Pull request merged**